lms/djangoapps/ccx/views.py

```diff
@@ -153,7 +153,13 @@ def edit_ccx_context(course, ccx, user):
     context['schedule'] = json.dumps(schedule, indent=4)
     context['save_url'] = reverse(
         'save_ccx', kwargs={'course_id': ccx_locator})
-    context['ccx_members'] = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
+
+    ccx_members = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
+
+    onlyStudents = lambda member: not CourseAccessRole.objects.filter(course_id=ccx_locator, user=member.user).exists()
```

Comment:
You make a DB query for each member in the list. Would it be better to fetch all user IDs from CourseAccessRole table and then filter ccx_members query with ~Q(id__in=ccx_staff_ids)? This way, only 2 DB queries are made regardless of ccx members size.